echopype/convert/set_groups_ek60.py

```diff
@@ -17,6 +17,36 @@
 class SetGroupsEK60(SetGroupsBase):
     """Class for saving groups to netcdf or zarr from EK60 data files."""
 
+    """
+    The sets beam_only_names, ping_time_only_names, and
+    beam_ping_time_names are used in set_groups_base and
+    in converting from v0.5.x to v0.6.0. The values within
+    these sets are applied to all Sonar/Beam_groupX groups.
+    """
```

Comment:
I'm not sure, but the presence of this string block here threw me off. It's like a docstring right after a docstring.

Also, I may be wrong here, but I don't think I've seen "stuff" placed here in a class, outside of `__init__`. I guess it works! But I would've expected it to be within `__init__`

Reply:
These are "class variables" that are not instance variables under `__init__`.

Reply:
Oh maybe you're just referring to the docstrings. Ha, I think there's some debate on where those go.

Reply:
Yes, I know this string looks out of place, I was using it for a multiline comment, but if you prefer, I can use #. In regards to the variables. I placed them before the `__init__` this makes them global variables of the class.

Reply:
Thanks, and sorry for my ignorance there. I like @leewujung 's description -- class variables rather than instance variables.

I think I'd vote for using #'s. Otherwise my first instinct was to read that block as part of the class docstring.

Reply:
> Thanks, and sorry for my ignorance there. I like @leewujung 's description -- class variables rather than instance variables.

Yes, that is a correct and good description. 

>  I think I'd vote for using #'s. Otherwise my first instinct was to read that block as part of the class docstring.

Sounds good, unless @leewujung has any objections, I will go ahead and change this.

Reply:
Go ahead and change this, and feel free to self-merge!